src/components/BookList.spec.js

```diff
@@ -61,12 +61,8 @@ describe('Book list page', () => {
             expect(rendered.container).toHaveTextContent('A Great Project');
         });
 
-        it('should show the shelf', () => {
-            expect(rendered.container).toHaveTextContent('Alpha');
-        });
-
         it('should show that the book is available', () => {
-            expect(rendered.container).toHaveTextContent('Available');
+            expect(rendered.queryByTitle('Available')).toBeInTheDocument();
```

Comment:
Would `.toBeVisible()` be a better test? `toBeInDocument()` tests that it's in the dom but doesn't tell anything of the user's experience AFAIK.